### DISCUSSION TITLE
Is there one rubocop _cop_ to ban all `%_`s?

### DISCUSSION THREAD

**Discussion by stdedos** [rubocop/rubocop#13452] [Q&A] [answered]
Is there one rubocop _cop_ to ban all `%_`s?

**Comment by dvandersluis** [comment, marked as answer]
There is no one cop to ban all of the [percent literals](https://docs.ruby-lang.org/en/master/syntax/literals_rdoc.html#label-Percent+Literals). Why do you want to do so?

You could have rubocop replace `%w` and `%i` by setting `EnforcedStyle: brackets` for `Style/WordArray` and `Style/SymbolArray`. The others are not governed by a cop.

**Reply by stdedos** [reply]
> Why do you want to do so?

Because some "find it hard to remember what each % does, and the code is harder to understand".

(tbh I _marginally_ get it, since someone wrote `%w[1 2 3 4 5 6]` assuming those are ints - and tested them via `(i % 2) == 0` - which RuboCop discovered as code smell. But that's it)
